tests/functional/test_confirm_email_address.py

```diff
@@ -1,37 +1,31 @@
-import pytest
 from flask import url_for
-
-email_depts = [('some.one@digital.cabinet-office.gov.uk',
-                'Government Digital Service')]
+from tests.functional.user_flows import on_email_confirm_fill_in_then_continue
+from tests.functional.user_flows import yes_selected, no_selected
 
 
 class When_on_confirm_email_with_yes_selected_and_valid_email(object):
 
-    @pytest.mark.parametrize("email_address,department", email_depts)
     def it_goes_to_department_confirm_when_continue_clicked(
-            self, live_server, browser, email_address, department):
-        browser.visit(url_for('main.request_email_address', _external=True))
+            self, live_server, browser):
 
-        browser.choose('email_known', 'yes')
-        browser.fill('email_address', email_address)
+        email_address = 'some.one@digital.cabinet-office.gov.uk'
+        department = 'Government Digital Service'
 
-        browser.find_by_css('form button').click()
+        on_email_confirm_fill_in_then_continue(
+            browser, yes_selected, email_address)
```

Comment:
Also, since these are essentially fixtures, you could write them like this to avoid passing `browser` around:
```python
@pytest.fixture
def set_email_known(browser):
    def do_set_email_known(known=True):
        value = 'yes' if known else 'no'
        browser.choose('email_known', value)
    return do_set_email_known
```
and then in the test
```python
def it_goes_to_department_confirm_when_continue_clicked(browser, set_email_known, submit_email):
    email_address = 'some.one@...'
    set_email_known()
    submit_email(email_address)
    ...
```